areas-of-d-usage.dd

```diff
@@ -322,4 +322,7 @@ Macros:
                 margin-top: -200px;
             }
         }
+        h2 {
```

Comment:
We already had this discussion on a different PR. In short
- inline CSS is _faster_
- we litter our main css file with such specific definitions (they are only needed on this page)
- if someone wants to modify this page, he directly has the CSS for it too
- we can't even put such definitions in the main css file because the body id contains a space